extensions/wikia/RTE/RTEMagicWord.class.php

```diff
@@ -19,7 +19,7 @@ public static function get(&$magicWords, $langCode) {
 		return true;
 	}
 
-	public static function remove( Parser $parser, string &$text, &$strip_state ): bool {
+	public static function remove($parser, &$text, &$strip_state) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove the unused function parameter "$parser". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)

[project-id]: wikia:app
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove the unused function parameter "$strip_state". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)

[project-id]: wikia:app